client/web/src/components/exit-node-selector.tsx

```diff
@@ -74,72 +76,88 @@ export default function ExitNodeSelector({
     >
       <div
         className={cx(
-          "p-1.5 rounded-md border flex items-stretch gap-1.5",
+          "rounded-md",
           {
-            "border-gray-200": none,
-            "bg-amber-600 border-amber-600": advertising,
-            "bg-blue-500 border-blue-500": using,
+            "bg-red-600": offline,
           },
           className
         )}
       >
-        <button
-          className={cx("flex-1 px-2 py-1.5 rounded-[1px]", {
-            "bg-white hover:bg-stone-100": none,
-            "bg-amber-600 hover:bg-orange-400": advertising,
-            "bg-blue-500 hover:bg-blue-400": using,
-            "cursor-not-allowed": disabled,
+        <div
+          className={cx("p-1.5 rounded-md border flex items-stretch gap-1.5", {
+            "border-gray-200": none,
+            "bg-yellow-300 border-yellow-300": advertising,
+            "bg-blue-500 border-blue-500": using,
+            "bg-red-500 border-red-500": offline,
```

Comment:
I think both of these could be true at the same time - not sure if this will mess with `cx`'s chosen classnames

Reply:
the last value seems to take effect, so this seems to work. But just to make sure that's not some browser-specific behavior, added `&& !offline` to the other cases.